silver/tests/spec/test_documents.py

```diff
@@ -14,21 +14,49 @@
 
 import json
 
+from django.test import override_settings
```

Comment:
I suggest to decouple django and rest framework imports from other 3-rd party libraries imports.